src/inference/src/dev/core_impl.cpp

```diff
@@ -1098,6 +1100,20 @@ std::shared_ptr<const ov::Model> ov::CoreImpl::apply_auto_batching(const std::sh
     return ov::details::apply_batch_affinity(model, deviceNameWithoutBatch);
 }
 
+void ov::CoreImpl::apply_rt_info(const ov::Plugin& plugin,
+                                 const std::shared_ptr<const ov::Model>& model,
+                                 ov::AnyMap& config) const {
+    if (util::contains(plugin.get_property(ov::supported_properties), ov::hint::activations_scale_factor)) {
+        if (model->has_rt_info({"runtime_options", "ACTIVATIONS_SCALE_FACTOR"})) {
+            if (config.find("ACTIVATIONS_SCALE_FACTOR") == config.end()) {
```

Comment:
"ACTIVATIONS_SCALE_FACTOR" -> `ov::hint::activations_scale_factor.name()`

Reply:
I updated it as you reviewed. Thank you.